zproject/backends.py

```diff
@@ -1148,7 +1150,9 @@ def get_or_build_user(self, username: str, ldap_user: _LDAPUser) -> Tuple[UserPr
         # We have valid LDAP credentials; time to create an account.
         full_name = self.get_mapped_name(ldap_user)
         try:
-            full_name = check_full_name(full_name)
+            full_name = check_full_name(
+                full_name_raw=full_name, user_profile=user_profile, realm=self._realm
+            )
```

Comment:
This `user_profile` parameter is always `None`.

Reply:
ya, I thought since a variable `user_profile` already exists, I would use it. I'll update it.

Reply:
No need to tweak, I've taken care of it.